README.md

```diff
@@ -8,6 +8,8 @@
 
 The shader translation library for the needs of [wgpu](https://github.com/gfx-rs/wgpu) and [gfx-rs](https://github.com/gfx-rs/gfx) projects.
 
+**Minimum Supported Rust Version:** `1.43.0`
```

Comment:
do you think we could have a real github badge for this instead of a note? I think I saw some in other projects

Reply:
I saw only generated with `shields.io`. Like this one: ![](https://img.shields.io/badge/rustc-1.43+-blue.svg)

Reply:
looks neat!

Reply:
Done. Rendered - https://github.com/Gordon-F/naga/blob/patch-2/README.md.